Update exchange-unified-messaging-online-migration-support.md (#3347)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Skype/SfBServer2019/plan/exchange-unified-messaging-online-migration-support.md

| AA | Service Features | Multi-language support | Language details here: https://docs.microsoft.com/en-us/microsoftteams/what-are-phone-system-auto-attendants | Y | Y    |
| AA | Service Features | Transfer to operator, CQ, or a user |  | Y | Y    |
| AA | Service Features | Transfer to PSTN number internally (DID RNL)  |  | Y | Y    |

> | AA | Service Features | Transfer to PSTN number externally  |  | Check out Known Issues section below | Y    |

Reviewer: ```suggestion
| AA | Service Features | Transfer to PSTN number externally  |  | See **Known issues** section below | Y    |
```

#### Known issues

**Auto Attendant Call Transfer to PSTN**

> Customers are encouraged to configure a temporarily workaround to fulfill the requirements of transferring an auto attendant call to an external PSTN number, or to an RGS instance. 

Reviewer: ```suggestion
Customers are encouraged to configure a temporary workaround to fulfill the requirements of transferring an auto attendant call to an external PSTN number, or to an RGS instance. 
```

> An issue was identified during quality assurance with the Transfer out to PSTN number feature, which is not going to be fixed in-time for customers to start migrating off Exchange UMO service before its scheduled retirement date of Feb 1st, 2020. As a workaround, administrators can transfer auto attendant callers to an on-premise virtual user with an active Call Forward setting to the desired PSTN phone number or RGS phone number. 

Reviewer: ```suggestion
An issue was identified during quality assurance with the Transfer out to PSTN number feature, which is not going to be fixed in time for customers to start migrating off Exchange UMO service before its scheduled retirement date of February 1, 2020. As a workaround, administrators can transfer auto attendant callers to an on-premises virtual user with an active Call Forward setting to the desired PSTN phone number or RGS phone number. 
```

> Expected Experience

Reviewer: ```suggestion
**Expected Experience**
```

> - Administrators do not need to license the virtual user, since this is a workaround solution 

Reviewer: ```suggestion
- Administrators do not need to license the virtual user, since this is a workaround solution.
```

> - Administrators can manipulate the caller ID that the PSTN receiver will see by assigning the desired number to the virtual user, or using the SBC digit manipulation capabilities 

Reviewer: ```suggestion
- Administrators can manipulate the caller ID that the PSTN receiver will see by assigning the desired number to the virtual user, or using the SBC digit manipulation capabilities.
```

> - PSTN Callers will not experience any delay during the call transfer, and they will continue to see the caller ID of the auto attendant after the transfer is successful  

Reviewer: ```suggestion
- PSTN Callers will not experience any delay during the call transfer, and they will continue to see the caller ID of the auto attendant after the transfer is successful.
```
